### ISSUE TITLE
Unable to start Language Server

### ISSUE THREAD

**Issue by alex-gilman** [CircleCI-Public/circleci-yaml-language-server#266]
Using the CircleCI extension in VSCode on Ubuntu 20.04.6, an error notification reading "Unable to start the CircleCI Language Server". CircleCI log shows

`[error] 2024-01-12T15:56:32.857Z Failed to start LSP server. Attempt 1: Error: Binary version mismatch: expected 0.9.2, got null`
`[error] 2024-01-12T15:56:39.820Z Failed to start LSP server. Attempt 2: Error: Binary version mismatch: expected 0.9.2, got null`
`[error] 2024-01-12T15:56:41.866Z Failed to start LSP server. Attempt 3: Error: Binary version mismatch: expected 0.9.2, got null`
`[error] 2024-01-12T15:56:44.228Z Failed to start LSP server. Attempt 4: Error: Binary version mismatch: expected 0.9.2, got null`
`[error] 2024-01-12T15:56:46.886Z Failed to start LSP server. Attempt 5: Error: Binary version mismatch: expected 0.9.2, got null`
`[error] 2024-01-12T15:56:51.890Z {}`

Similarly after executing the "Update CircleCI Language Server" command.

Extension: v2.3.0

VSCode:
- Version: 1.85.1
- Commit: 0ee08df0cf4527e40edc9aa28f4b5bd38bbff2b2
- Date: 2023-12-13T09:47:11.635Z
- Electron: 25.9.7
- ElectronBuildId: 25551756
- Chromium: 114.0.5735.289
- Node.js: 18.15.0
- V8: 11.4.183.29-electron.0
- OS: Linux x64 5.15.0-91-generic

**Comment by loderunner** [comment]
Thanks for reporting this. We've recently identified a bug in this area. I have a question: are you perhaps running VS Code inside a dev container?

**Comment by alex-gilman** [comment]
No, VS Code is installed on my system as an apt package.

**Comment by abdelDriowya** [comment]
Hi @alex-gilman do you still have this issue ? 
Do you have some setup evolving ssh into a remote machine or something similar ?

**Comment by alex-gilman** [comment]
Yes, I still have the issue. There is nothing complicated about my setup. I am running VS Code (installed as an apt package) directly on my machine.

**Comment by abdelDriowya** [comment]
Thanks for your answer, can you run the following commands
`ls -l ~/.local/share/CircleCI-language-server`
`md5  ~/.local/share/CircleCI-language-server/circleci-yaml-language-server.tmp`
`md5  ~/.local/share/CircleCI-language-server/circleci-yaml-language-server`
`~/.local/share/CircleCI-language-server/circleci-yaml-language-server --version`
`~/.local/share/CircleCI-language-server/circleci-yaml-language-server.tmp --version`
and share with us the output of these commands

**Comment by alex-gilman** [comment]
`~$ ls -l ~/.local/share/CircleCI-language-server`
`total 34636`
`-rwxr-xr-x 1 agilman agilman 17876712 Nov 29 09:20 circleci-yaml-language-server*`
`-rwxr-xr-x 1 agilman agilman 17421792 Jan 23 09:06 circleci-yaml-language-server.tmp*`
`-rw-rw-r-- 1 agilman agilman    78608 Nov 29 09:20 schema.json`
`-rw-rw-r-- 1 agilman agilman    78608 Jan 23 09:06 schema.json.tmp`

`~$ md5sum ~/.local/share/CircleCI-language-server/circleci-yaml-language-server.tmp`
`722a45ab447a26bd2018fe0b7009b940`

`~$ md5sum ~/.local/share/CircleCI-language-server/circleci-yaml-language-server`
`54bf17b803097eb17f6fce00444caef4`

`~$ ~/.local/share/CircleCI-language-server/circleci-yaml-language-server --version`
`0.9.1`

`~$ ~/.local/share/CircleCI-language-server/circleci-yaml-language-server.tmp --version`
`/home/agilman/.local/share/CircleCI-language-server/circleci-yaml-language-server.tmp: /lib/x86_64-linux-gnu/libc.so.6: version 'GLIBC_2.32' not found (required by /home/agilman/.local/share/CircleCI-language-server/circleci-yaml-language-server.tmp)`
`/home/agilman/.local/share/CircleCI-language-server/circleci-yaml-language-server.tmp: /lib/x86_64-linux-gnu/libc.so.6: version 'GLIBC_2.34' not found (required by /home/agilman/.local/share/CircleCI-language-server/circleci-yaml-language-server.tmp)`

Those last errors prompted me to check my `libc6` installation. It is at version 2.31-0ubuntu9.14, and no updates appear to be available.

**Comment by amohield** [comment]
> Thanks for reporting this. We've recently identified a bug in this area. I have a question: are you perhaps running VS Code inside a [dev container](https://code.visualstudio.com/docs/devcontainers/containers)?

Hi @loderunner, is there a solution if I am using a dev container?
